Makefile

```diff
@@ -218,29 +217,11 @@ update-all-release-numbers:
 release-docs:
 	go vet ./cmd/release/docs
 	go run ./cmd/release/docs/main.go \
-		--branch=$(RELEASE_BRANCH) \
-		--usePrevReleaseManifestForComponentTable=$(USE_PREV_RELEASE_MANIFEST) \
-		--openPR=$(OPEN_PR) \
```

Comment:
Does removing this flag remove the ability to dry run the release docs without submitting PR? Did this flag offer that in the first place?

Reply:
The only reason this flag was needed was because I'd set it to false for making the docs so it wouldn't automatically check out the previous branch at at the end and I could manually add the changelog stuff. But with your pending changes, this isn't needed. 
And running the command doesn't automatically open a PR; it just opens an unsubmitted PR in a new tab. So there's not real risk in running these. So I'd really prefer if it was gone